### ISSUE TITLE
Incoming events not processed after renaming entity

### ISSUE THREAD

**Issue by weitzelb** [maciej-or/hikvision_next#61]
Everything worked as expected until I renamed the entities. For readability and usability I wanted to re-name the entities to something more meaningful e.g. "binary_sensor.ds_2cd2387g2_lu######_1_fielddetection" to "binary_sensor.frontyard_1_fielddetection".

Now once an event is triggered from the camera it is not processed anymore. A log message is shown.

`Source: custom_components/hikvision_next/notifications.py:47
Cannot process incoming event Entity not found binary_sensor.ds_2cd2387g2_lu######_1_fielddetection`

Seemingly some internal references are still set to the original entity id.

I just glanced through the code and I was wondering if it was related to
https://github.com/maciej-or/hikvision_next/blob/1f6d40ee62028d9834ac2cb82bcd70c805c0f668/custom_components/hikvision_next/notifications.py#L146

Most other instances you used the unique id but in this case you are re-building the original entity id based on the camera information.

**Comment by maciej-or** [comment]
Hi, as far changing entity ids is forbidden, explained here: https://github.com/maciej-or/hikvision_next/issues/32#issuecomment-1523936473

**Comment by weitzelb** [comment]
Thanks for the hint. I didn't regard that issue earlier since the name stated a different topic. 
I kept on reading your code and doing some tests. I then noticed that it's because of the traceability of the incoming event. To be neccesarilly maped to something known (which the unique id is not at that point).

I am not too faimilar with the development of Home Assistant integrations, honestly. First time I actually looked into it. So just a random thought: There is no "metadata" that could be written to the entity or device containing the original camera identifier? Upon a request arriving select the entity containing that specific "metadata"?
